src/libraries/System.Linq.Expressions/tests/Dynamic/InvokeMemberBindingTests.cs

```diff
@@ -335,8 +331,7 @@ private class ManyOverloads
                 int arg10) => 11;
         }
 
-        // We're not testing compilation, but we do need Reflection.Emit for the test
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotLinqExpressionsBuiltWithIsInterpretingOnly))]
+        [Fact]
```

Comment:
This was probably hitting some .NET Native specific issues with dynamic keyword. I don't see it using Ref.Emit anywhere so leaving like this.